enaml/core/code_generator.py

```diff
@@ -383,6 +430,9 @@ def try_squash_raise(self):
         the code, rather than any function called by the code.
 
         """
+        if PY311:
+            yield
+            return  # TODO: Is this still possible?
```

Comment:
Using TryBegin and TryEnd pseudo instructions this should be feasible. I can give it a try but I may prioritize getting the pegen branch in first since we need it to.